README.md

```diff
@@ -66,12 +66,18 @@ Just run the following command at the root of your project and answer questions:
 npx readme-md-generator
 ```
 
-OR use default values for all questions (`-y, --yes`):
+Use default values for all questions (`-y, --yes`):
 
 ```sh
 npx readme-md-generator -y
 ```
 
+Use your custom template (`-p, --path`) (currently we support ejs template only, check our example [here](https://raw.githubusercontent.com/kefranabg/readme-md-generator/master/templates/default.md)):
```

Comment:
Unfortunately, we can't update the documentation for now 😄
I'll update the documentation when I'll publish the new release (`4.2.0`) on npm.

If we update the doc in this PR this will be displayed for all users but they won't be able to use the `-p` option as the published version on npm is still the old one

Reply:
Got it. Let me revert that commit.

Reply:
@yannbertrand I removed that commit.